content/chat/moderation/index.textile

```diff
@@ -27,6 +31,22 @@ This approach provides additional safety guarantees, but may come at the cost of
 
 h3(#after-publish). After publish
 
-When using after publish moderation, a message is published as normal, but is forwarded to a moderation engine after the fact. This allows you to avoid the latency penalty of vetting content prior to publish, at the expense of bad content being visible in the Chat room (at least briefly). Many automated moderation solutions are able to process and delete offending messages within a few seconds of publication.
+When using after publish moderation, a message is published as normal, but is forwarded to a moderation engine after the fact. This enables you to avoid the latency penalty of vetting content prior to publish, at the expense of bad content being visible in the Chat room (at least briefly). Many automated moderation solutions are able to process and delete offending messages within a few seconds of publication.
```

Comment:
```suggestion
When using after publish moderation, a message is published as normal, but is forwarded to a moderation engine after the fact. This enables you to avoid the latency penalty of vetting content prior to publish, at the expense of bad content being visible in the chat room (at least briefly). Many automated moderation solutions are able to process and delete offending messages within a few seconds of publication.
```

Reply:
186d374b9